tests/openthread/README.md

```diff
@@ -1,36 +1,43 @@
 ## OpenThread on RIOT
 
-This test demonstrates the [OpenThread](https://github.com/openthread/openthread) stack running on RIOT. When flashed,
-it will initialize the OpenThread Command Line Interface for interacting with the stack.
+This test demonstrates how to use OpenThread stack from [OpenThread](https://github.com/openthread/openthread) on
+RIOT. The [Command Line Interface](https://github.com/openthread/openthread/blob/master/examples/apps/cli/README.md) of
+OpenThread was ported. Please check the [full
+documentation](https://github.com/openthread/openthread/blob/master/src/cli/README.md) of the CLI for usage information.
+
+You can choose to compule a FTD or MTD with :
+`make BOARD=<target> clean all flash OPENTHREAD_TYPE=mtd/ftd`
+- Minimal Thread Device (MTD) does not have router functionality compiled in.An MTD may or may not be sleepy.
+- Full Thread Device (FTD). A Full Thread Device has router functionality compiled in.
 
 ## Quick usage
 
-To test OpenThread on RIOT, you can do the following:
+With RIOT port, a node is auto-setup and ready to communicate with this configuration:
+PANID: 0x1234
+CHANNEL: 26
 
-1. Flash nodes with `make BOARD=<target> clean all flash`
-2. Write `panid 0x1234`, `ifconfig up` then `thread start` on one node.
-3. Check the state of the node with `state`. In the beggining should be `detached`, but after some seconds it should
-   become `leader`
-4. Write `panid 0x1234`, `ifconfig up` then `thread start` on another node.
-The second node should become `child` or `router` if there's a leader.
-5. Get the mesh IP address of a node with `ipaddr`.
+You can edit the auto-setup in the Makefile
+`#CFLAGS += -DOPENTHREAD_PANID=0xbeef -DOPENTHREAD_CHANNEL=11`
 
+To test OpenThread in RIOT, you can do the following:
+
+1. Flash nodes with `make BOARD=<target> clean all flash OPENTHREAD_TYPE=mtd/ftd`
```

Comment:
Split this in 2 lines: one for each type of build (mtd and ftd). I can imagine brainless users try the exact command and falling the ftd case.